flask_table/table.py

```diff
@@ -163,4 +168,18 @@ def create_table(name=str('_Table'), base=Table):
         # Then assume that what we have is a single class, so make it
         # into a 1-tuple.
         base = (base,)
-    return TableMeta(name, base, {})
+
+    attr_dict = {}
+    done = False
+    for base_class in base:
+        if not done:
+            for option in options:
+                if option in base_class.__dict__:
+                    attr_dict[option] = options[option]
+                    if len(attr_dict) == len(options):
+                        done = True
+                        break
+        else:
+            break
```

Comment:
Can we not just pass `options` straight into the 3rd argument of `TableMeta`? What breaks if you make this change?

Or to put it another way, what does this bit of code fix?